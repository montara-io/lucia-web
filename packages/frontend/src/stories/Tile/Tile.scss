@import '../../styles/colors';

.m-tile {
  border: 2px solid $gray240;
  height: 3em;
  display: flex;
  justify-content: center;

  &:hover {
    box-shadow: 0 3px 6px #00000029 !important;
    opacity: 1;
    background: $white 0 0 no-repeat padding-box;
  }
}
