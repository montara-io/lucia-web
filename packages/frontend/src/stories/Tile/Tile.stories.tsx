import React from 'react';
import Tile from './Tile';

export default {
  title: 'Components/Tile',
  component: Tile,
};

export const Basic = (args) => <Tile {...args}/>;

Basic.args = {};

Basic.parameters= {
  jest: ['Tile.test.tsx']
}
