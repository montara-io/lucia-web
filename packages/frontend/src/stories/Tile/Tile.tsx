import classNames from 'classnames';
import React from 'react';
import './Tile.scss';

export type TileProps = {
  children?: any;
  className?: string;
};

const Tile = (props) => {
  return <div className={classNames('m-tile', props.className)}>{props.children}</div>;
};

export default Tile;
