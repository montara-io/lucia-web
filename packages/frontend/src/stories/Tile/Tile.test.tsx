import React from 'react';
import { render } from '@testing-library/react';
import Tile from './Tile';

describe('Tile Component', () => {
  test('it should match the snapshot', () => {
    const { asFragment } = render(<Tile />);
    expect(asFragment()).toMatchSnapshot();
  });
});
