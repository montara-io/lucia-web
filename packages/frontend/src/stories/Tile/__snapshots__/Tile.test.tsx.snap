// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile Component it should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="m-tile"
  />
</DocumentFragment>
`;
